stock_mts_mto_rule/__manifest__.py

```diff
@@ -8,6 +8,7 @@
  'category': 'Warehouse',
  'summary': 'Add a MTS+MTO route',
  'depends': ['stock',
+             'mrp',
```

Comment:
What is the focus to split the modules if you still need mrp here

Reply:
my :bat: